.github/pull_request_template.md

```diff
@@ -2,11 +2,11 @@
 What does this PR do?
 
 ### TODO only if you're a HashiCorp employee
-- [ ] **Labels:** If this PR is the CE portion of an ENT change, and that ENT change is
-  getting backported to N-2, use the new style `backport/ent/x.x.x+ent` labels
-  instead of the old style `backport/x.x.x` labels.
-- [ ] **Labels:** If this PR is a CE only change, it can only be backported to N, so use
-  the normal `backport/x.x.x` label (there should be only 1).
+- [ ] **CE Labels:** If this PR is a CE-only change, you should only backport to N, so use
+  the `backport/x.x.x` label, not the enterprise labels (there should be only 1).
+- [ ] **Enterprise Labels:** If this PR is the CE portion of an ENT change, you should backport
+  to N, N-1, and N-2. Use the enterprise-only `backport/ent/x.x.x+ent` labels for N-1 and N-2, and the `backport/x.x.x` label for N.
+- [ ] **LTS Labels**: If this PR contains a fix for a critical security vulnerability or [severity 1](https://www.hashicorp.com/customer-success/enterprise-support) bug, it will also need to be backported to the current LTS branches. If an [LTS version](https://developer.hashicorp.com/vault/docs/enterprise/lts#why-is-there-a-risk-to-updating-to-a-non-lts-vault-enterprise-version) of Vault is further back than N-2, be sure to add the appropriate enterprise label (`backport/ent/x.x.x+ent`) for that branch.
```

Comment:
It'd be great if we had a list of current LTS versions somewhere, so that it's easy enough to do. Currently this checkbox makes it hard for me to actually know what's LTS. 

LTS versions so far are 1.N where (N % 3 = 1) -- it might be useful to either include that formula or have a link (or even list that we update in the future) to the current list of LTS releases.

Reply:
This is why I linked to the section of the public docs that I did, on the “LTS versions” link. It is the only public place I know of that shows that image of the LTS windows. If there’s a different section of that doc that’s more clear in your opinion, I can link to that, but I liked the calendar image personally.

Reply:
It takes a little to scroll down and parse the table, it'd be great if there was just a list of the LTS versions themselves so that we don't need to follow a link. Alternatively, we could just list directly, since we don't expect these to change much.

Or, actually -- since we clean up old labels, let's imagine a future where we're releasing 1.20 -- we'll have a backport to 1.16 label but not a backport to 1.17 label, so I don't actually need to know that 1.16 is an LTS or not, all I need to know is that I should add every backport label, not just N-2. Since we clean up old labels this should work seamlessly!

I think that might be even clearer: rather than listing out or asking the PR author to find out what the LTS versions are, instead, if it's LTS-backportable, they should tick every available backport label instead of N-2.

Reply:
How about:

> [ ] Backport Labels: If this PR is in the CE repo and needs to be backported, to backport to N, use the backport/x.x.x label, not the enterprise labels. If this is in the enterprise repo and to backport to N-1 or further in CE, use the enterprise labels. If this fixes a critical security vulnerability or a [severity 1](https://www.hashicorp.com/customer-success/enterprise-support) bug and should be backported to LTS releases, use every available enterprise label.